pkg/operators/clustermanager/controllers/clustermanagercontroller/clustermanager_runtime_reconcile.go

```diff
@@ -29,6 +29,7 @@ var (
 		"cluster-manager/management/cluster-manager-registration-webhook-deployment.yaml",
 		"cluster-manager/management/cluster-manager-work-webhook-deployment.yaml",
 		"cluster-manager/management/cluster-manager-placement-deployment.yaml",
+		"cluster-manager/management/cluster-manager-manifestworkreplicaset-deployment.yaml",
```

Comment:
should we check the featureGate EnableReplicaSetManager to determine if install the manifestworkreplicaset manifests like  https://github.com/open-cluster-management-io/registration-operator/blob/547fc1482e257e7eb8abfcd13c456513837d5574/pkg/operators/clustermanager/controllers/clustermanagercontroller/clustermanager_runtime_reconcile.go#L106 ?